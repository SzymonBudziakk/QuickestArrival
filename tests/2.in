19 15
*#####*EE.FF*#####*
.AA...#.....#...BB.
......#######......
...AAAA..#..BBBB...
....*#########*....
GG.......#.......II
*........#........*
###################
*........#........*
HH.......#.......JJ
....*#########*....
...DDDD..#..CCCC...
.........#.........
.DD......#......CC.
*#################*
0
130
AA BB 1
AA CC 1
AA DD 1
AA EE 1
AA FF 1
AA GG 1
AA HH 1
AA II 1
AA JJ 1
AA AAAA 1
AA BBBB 1
AA CCCC 1
AA DDDD 1
BB BB 1
BB CC 1
BB DD 1
BB EE 1
BB FF 1
BB GG 1
BB HH 1
BB II 1
BB JJ 1
BB AAAA 1
BB BBBB 1
BB CCCC 1
BB DDDD 1
CC BB 1
CC CC 1
CC DD 1
CC EE 1
CC FF 1
CC GG 1
CC HH 1
CC II 1
CC JJ 1
CC AAAA 1
CC BBBB 1
CC CCCC 1
CC DDDD 1
DD BB 1
DD CC 1
DD DD 1
DD EE 1
DD FF 1
DD GG 1
DD HH 1
DD II 1
DD JJ 1
DD AAAA 1
DD BBBB 1
DD CCCC 1
DD DDDD 1
EE BB 1
EE CC 1
EE DD 1
EE EE 1
EE FF 1
EE GG 1
EE HH 1
EE II 1
EE JJ 1
EE AAAA 1
EE BBBB 1
EE CCCC 1
EE DDDD 1
FF BB 1
FF CC 1
FF DD 1
FF EE 1
FF FF 1
FF GG 1
FF HH 1
FF II 1
FF JJ 1
FF AAAA 1
FF BBBB 1
FF CCCC 1
FF DDDD 1
GG BB 1
GG CC 1
GG DD 1
GG EE 1
GG FF 1
GG GG 1
GG HH 1
GG II 1
GG JJ 1
GG AAAA 1
GG BBBB 1
GG CCCC 1
GG DDDD 1
HH BB 1
HH CC 1
HH DD 1
HH EE 1
HH FF 1
HH GG 1
HH HH 1
HH II 1
HH JJ 1
HH AAAA 1
HH BBBB 1
HH CCCC 1
HH DDDD 1
II BB 1
II CC 1
II DD 1
II EE 1
II FF 1
II GG 1
II HH 1
II II 1
II JJ 1
II AAAA 1
II BBBB 1
II CCCC 1
II DDDD 1
JJ BB 1
JJ CC 1
JJ DD 1
JJ EE 1
JJ FF 1
JJ GG 1
JJ HH 1
JJ II 1
JJ JJ 1
JJ AAAA 1
JJ BBBB 1
JJ CCCC 1
JJ DDDD 1
