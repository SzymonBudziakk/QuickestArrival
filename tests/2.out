23 EE FF
32 EE
32 EE
6
16 EE
26 EE
26 EE
26 EE
26 EE
18 EE
18 EE
24 EE
24 EE
0
32 FF
32 FF
16 FF
6
26 FF
26 FF
26 FF
26 FF
18 FF
18 FF
24 FF
24 FF
32 FF
0
18
26
26
26
26
26
26
24
24
18
18
32 FF
18
0
26
26
26
26
26
26
24
24
18
18
16 FF
26
26
0
10
20
20
20
20
12
12
18
18
6
26
26
10
0
20
20
20
20
12
12
18
18
26 FF
26
26
20
20
0
2
20
20
18
18
18
18
26 FF
26
26
20
20
2
0
20
20
18
18
18
18
26 FF
26
26
20
20
20
20
0
2
18
18
18
18
26 FF
26
26
20
20
20
20
2
0
18
18
18
18
